app/test/testmerginapi.cpp

```diff
@@ -61,11 +67,136 @@ void TestMerginApi::testDownloadProject()
     ProjectList projects = mMerginProjectModel->projects();
     Q_ASSERT(!mMerginProjectModel->projects().isEmpty());
     mProjectName = mMerginProjectModel->projects().at(0)->name;
-    qDebug() << "TestMerginApi::testListProjectFinished PASSED";
+    qDebug() << "TestMerginApi::testDownloadProject PASSED";
+}
+
+void TestMerginApi::testCreateProjectTwice()
+{
+    qDebug() << "TestMerginApi::testCreateProjectTwice START";
+    mProjectName = "TEMPORARY_TEST_PROJECT";
+    bool containsTestProject = false;
+    ProjectList projects = getProjectList();
+    for (std::shared_ptr<MerginProject> project: projects) {
+        if (project->name == mProjectName) containsTestProject = true;
+    }
```

Comment:
Since we use this piece of code multiple times, let's put it into a function we can reuse, e.g. `bool hasProject(projectName, projects)`